packages/lib-webpack/src/webpack/DynamicRemotePlugin.ts

```diff
@@ -84,10 +90,22 @@ export type DynamicRemotePluginOptions = Partial<{
    * See {@link PluginEntryCallbackSettings} properties and their defaults.
    */
   entryCallbackSettings: PluginEntryCallbackSettings;
+
+  /**
+   * Select module injection type
+   * 
+   * jsonp is consumed via pre-defined callback. By default __load_plugin_entry__(..)
+   * var is assigned to global variable name
+   * 
+   * Default value: jsonp
+   */
+  libraryType: 'jsonp' | 'var'
```

Comment:
I do personally tend to adopt dependency terminology. It makes it easier to find the original docs if you'd like to learn more. But I do understand that in this case, it might not make sense at first glance. I think we can rename it, but I will add a link to the webpack docs.